ui/store/actions.test.js

```diff
@@ -1231,7 +1237,79 @@ describe('Actions', () => {
         },
       ];
 
-      await store.dispatch(actions.setRpcTarget());
+      await store.dispatch(actions.setActiveNetwork());
+      expect(store.getActions()).toStrictEqual(expectedActions);
+    });
+  });
+
+  describe('#editAndSetNetworkConfiguration', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('calls editAndSetNetworkConfiguration', async () => {
```

Comment:
Nit: This description doesn't seem right. It's not calling `editAndSetNetworkConfiguration`, that's the function under test.
```suggestion
    it('Removes then re-adds the given network configuration', async () => {
```

Reply:
🤔 🤣